src/templates/static.tsx

```diff
@@ -190,26 +245,55 @@ const StaticPage = ({
   const absoluteEditPath = `${editContentUrl}${relativePath}`
 
   return (
-    <Container>
+    <Box w="full">
       {showPostMergeBanner && (
         <PostMergeBanner
           translationString={postMergeBannerTranslationString!}
         />
       )}
-      <Page dir={isRightToLeft ? "rtl" : "ltr"}>
+      <Flex
+        justifyContent="space-between"
+        w="full"
+        mx="auto"
+        mb={16}
+        p={8}
+        pt={{ lg: 16 }}
+        dir={isRightToLeft ? "rtl" : "ltr"}
+      >
         <PageMetadata
           title={mdx.frontmatter.title}
           description={mdx.frontmatter.description}
         />
-        <ContentContainer>
+        <Box
+          as="article"
+          maxW="container.md"
+          w="full"
+          sx={{
+            ".featured": {
+              pl: 4,
+              ml: -4,
+              borderLeft: "1px dotted",
+              borderLeftColor: "primary",
+            },
+
+            ".citation": {
+              p: {
+                color: "text200",
+              },
+            },
+          }}
+        >
           <Breadcrumbs slug={slug} />
-          <LastUpdated
-            dir={isLangRightToLeft(language as Lang) ? "rtl" : "ltr"}
+          <Text
+            color="text200"
+            dir={isLangRightToLeft(intl.locale as Lang) ? "rtl" : "ltr"}
```

Comment:
Looks like after merging of the dev, the two lines for the local strings did not get updated. (To go with line 220)

```suggestion
            dir={isLangRightToLeft(language as Lang) ? "rtl" : "ltr"}
```